README.md

```diff
@@ -50,6 +53,21 @@ make dev
 
 The application will then sleep for 24 hours.
 
+<details>
+
+<summary>Rebuilding the container</summary>
+
+Code is attached to the containers as volumes so most updates are visible without a need to rebuild the container.
+However, changes to NPM dependencies, for example, require a rebuild. So you may need to execute
```

Comment:
Worth having make targets that hide this complexity?

Reply:
I haven't been making it a dependency as a `build` is not needed in most cases. (Due to Docker caches it's not that slow, but still...)